src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -15,12 +15,12 @@ import { cleanUpHtml, processHtml } from '../../../utils/html';
 import { getAnchorProperty, setAnchorProperty } from './anchor';
 import { getAttributesProperty } from './attributes';
 import { URL_ATTR_TAGS, TARGET_ATTR_TAGS } from '../../../dom-processor';
-import { process as processStyle, cleanUp as cleanUpStyle } from '../../../../processing/style';
+import styleProcessor from '../../../../processing/style';
```

Comment:
We can't import separate methods of object because they will not get context in this case